src/cascade/executor/session_options.py

```diff
@@ -23,15 +23,17 @@ def make_options(ev_settings):
     options["ode_step_size"] = ev_settings.model.ode_step_size
     if ev_settings.model.additional_ode_steps:
         options["age_avg_split"] = " ".join(str(ea) for ea in ev_settings.model.additional_ode_steps)
-    if hasattr(ev_settings, "random_seed") and ev_settings.random_seed:
-        options["random_seed"] = ev_settings.random_seed
+    if hasattr(ev_settings.model, "random_seed") and ev_settings.model.random_seed:
```

Comment:
If this field is in the form configuration it should always be present on the settings objects, even if it wasn't set by the users. If you are checking for whether the user supplied it then use `ev_settings.model.is_field_unset("random_seed")`

Also, this check does the wrong thing if the user specifies a random_seed of `0` (which they shouldn't but you know, technically....)

Reply:
Alec, how does this work for something like zero_sum_random, that's a list? I don't know how to use is_field_unset() for this because lists don't have that method. If someone sets zero sum random to something and then wants to unset it... that's not a use case here, is it?

Reply:
Oh, I'm doing it wrong. It's on the form.